DEV/RootTabBarController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  RootTabBarController.swift
+//  DEV
+//
+//  Created by Jacob on 8/27/18.
+//  Copyright © 2018 DEV. All rights reserved.
+//
+
+import UIKit
+
+class RootTabBarController: UITabBarController {
+    //will change this to DevWebViewController
+    private var viewControllerList : [UIViewController] = []
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.delegate = self
+        
+        viewControllerList.append(storyboard!.instantiateViewController(withIdentifier: "Home"))
+        viewControllerList.append(storyboard!.instantiateViewController(withIdentifier: "Search"))
+        viewControllerList.append(storyboard!.instantiateViewController(withIdentifier: "Notifications"))
+        viewControllerList.append(storyboard!.instantiateViewController(withIdentifier: "Connect"))
+        viewControllerList.append(storyboard!.instantiateViewController(withIdentifier: "Self"))
+        viewControllers = viewControllerList
```

Comment:
You're right these aren't necessary, I placed them there kind of as placeholders once we move from using storyboard files.  Maybe I should have commented them out.  I wasn't sure what to include for this PR and what not because the purpose of the RootTabBarController was to move away from storyboards, but we are still using storyboards.  

I wouldn't mind changing all of the VCs to use DevWebVC instead of the storyboard, I did that originally with these changes but then converted it back to use storyboards.  All I need is the green light to make that move lol